### PR TITLE
802.1Qav settings support

### PR DESCRIPTION
**Pull request by jukkar**

### CODE REVIEW

File: include/net/ethernet.h [hunk 1]
```diff
@@ -75,13 +75,26 @@ enum ethernet_hw_caps {
 
 	/** IEEE 802.1AS (gPTP) clock supported */
 	ETHERNET_PTP			= BIT(8),
+
+	/** IEEE 802.1 Qav supported */
```
Comment (pfalcon) [inline include/net/ethernet.h]: I wish this comment was like one above (for 802.1AS): "IEEE 802.1 Qav (&lt;something>) supported"

**Review by pfalcon — APPROVED** [review]
Suggestion to explain somewhere what the heck 802.1Qav is, otherwise looks good.

**Comment by tgorochowik** [comment]
@pfalcon - changed the comment to: `/** IEEE 802.1Qav (credit-based shaping) supported */` Is that okay? Or do you think that a proper doc is needed?

**Comment by pfalcon** [comment]
@tgorochowik:

> changed the comment to: /** IEEE 802.1Qav (credit-based shaping) supported */ Is that okay? 

Thanks, I appreciate that! That's enough, someone needed more info can surely google it up. But having basic info inline when someone browses thru things is very helpful.

**Pull request merged**